src/main/java/mate/academy/bookstore/handler/ResponseData.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.bookstore.handler;
+
+import java.util.List;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public abstract class ResponseData {
```

Comment:
This class should contain similar fields for SuccessResponseData and ErrorResponse data, fields data will be only in SuccessResponseData. No need to add it here to make it null